openlibrary/macros/BookPreview.html

```diff
@@ -5,7 +5,7 @@
     <a class="cta-btn cta-btn--preview"
       data-iframe-src="https://archive.org/details/$ocaid?view=theater&wrapper=false"
       data-iframe-link="https://archive.org/details/$ocaid"
-      $:analytics_attr('Preview') href="#bookPreview">$_('Preview') <span>$_('Only')</span></a>
+      $:analytics_attr('Preview') href="#bookPreview">$_('Preview Only')</a>
```

Comment:
Oh wait I see, it's rendered without "only" on the books page 🙃 Can you add an `display_only=False` parameter, and switch between the two to display either `Preview` or `Preview Only`? And then remove this CSS as well:

https://github.com/internetarchive/openlibrary/blob/02a343edd25dc696de224ebe6dcf33402ba707ed/static/css/components/read-panel.less#L70-L73

And update these two spots to have `show_only=True`

https://github.com/internetarchive/openlibrary/blob/99cd085892223b7b249971060c7fe5985588824e/openlibrary/macros/LoanStatus.html#L68-L76